lib/ansible/cli/galaxy.py

```diff
@@ -485,9 +485,23 @@ def add_install_options(self, parser, parents=None):
             args_kwargs['help'] = 'Role name, URL or tar file'
             ignore_errors_help = 'Ignore errors and continue with the next specified role.'
 
+        if self._implicit_role:
+            # installing both roles and collections
+            description_text = (
+                'Install roles and collections from file(s), URL(s) or Ansible '
+                'Galaxy to the first entry in the config COLLECTIONS_PATH for collections '
+                'and first entry in the config ROLES_PATH for roles.'
+            )
```

Comment:
This could mention that if `-p`/`--roles-path` is provided only roles are installed. If there are collections in the requirements.yml too, a warning is given about the potentially ambiguous option, and that collections won't be installed.

> the ansible-galaxy install command has a --roles-path CLI option, but no collections path option

None of the collection-specific CLI install options are supported, only collection options that are also supported by roles. Role-specific options are supported for backwards compatibility because `ansible-galaxy install` handled roles originally.

If a `--collections-path` option is added, `-p` should probably be removed as a valid alias for `--roles-path`, since it's also an alias normally for `--collections-path`.

Reply:
tempted to say just use `--install-path` and even if people see the 'wrong' help .. it will be right!